lib/src/upgrade_messages.dart

```diff
@@ -247,6 +247,9 @@ class UpgraderMessages {
         message =
             '{{appName}}有新的版本！您拥有{{currentInstalledVersion}}的版本可更新到{{currentAppStoreVersion}}的版本。';
         break;
+      case 'ro':
```

Comment:
@SK1n Thanks for your PR. The language codes like 'ro' are in English and are alphabetized, so 'ro' should be right below 'ps'. Can you move the changes for 'ro' to the right place?